roles/etcd/tasks/upgrade_rpm.yml

```diff
@@ -10,6 +10,10 @@
 
 - name: Verify cluster is healthy pre-upgrade
   command: "{{ etcdctlv2 }} cluster-health"
+  register: _u_r_cluster_health
+  retries: 6
```

Comment:
This should be retries 30, delay 6, 30 seconds is too long to wait.

Reply:
updated and removed registered variable prefixes